### DISCUSSION TITLE
因工作繁忙，本项目不再支持微信群答疑，仅创建 issue 讨论

### DISCUSSION THREAD

**Discussion by johnnyise** [NetEase/tango#7] [Announcements]
## 👋 大家好 Welcome!

  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

### 文档站点 Documentation Site

https://netease.github.io/tango-site/

**Comment by FKLam** [comment]
微信群不存在了，麻烦更新下二维码，谢谢。

**Reply by wwsun** [reply]
已更新，3.7前有效

**Comment by wangminqinag** [comment]
微信群不存在了，麻烦更新下二维码，谢谢

**Reply by wwsun** [reply]
已更新，3.7前有效

**Comment by careteenL** [comment]
刚看到，失效了😂， 麻烦更新下二维码，谢谢

**Reply by ccloli** [reply]
已更新，3.22 前有效

**Comment by pigjs** [comment]
失效了，麻烦更新一下二维码，或者提供一下微信号，谢谢

**Reply by ccloli** [reply]
已更新，0429 前有效

**Comment by ghost** [comment]
失效了，麻烦更新一下二维码

**Reply by professor-jin** [reply]
+1

**Comment by SKYDA** [comment]
失效了，麻烦更新一下二维码

**Reply by wwsun** [reply]
已更新

**Comment by luoheix** [comment]
失效了，麻烦更新一下二维码

**Comment by wuf-Gavin** [comment]
失效了，麻烦更新一下二维码

**Comment by Tkzc0811** [comment]
失效了，麻烦更新一下二维码

**Comment by zhanghaosysu** [comment]
可以更新下微信二维码吗，求加群

**Comment by xiaoqiangsdl** [comment]
+1 求微信群

**Comment by 1303106965** [comment]
+1 求群

**Comment by qinglinguo** [comment]
求微信群
